src/trees/MWNode.cpp

```diff
@@ -49,6 +49,7 @@ MWNode<D>::MWNode()
         : tree(nullptr)
         , parent(nullptr)
         , squareNorm(-1.0)
+        , maxSquareNorm(-1.0)
```

Comment:
`maxWSquareNorm` should also be initialized?

Reply:
done